extern/Makefile.in

```diff
@@ -1,8 +1,9 @@
-# Makefile.in generated by automake 1.14.1 from Makefile.am.
+# Makefile.in generated by automake 1.11.1 from Makefile.am.
```

Comment:
How come we are now using an older version of automake? I guess as this is external code we (as in sherpa) doesn't care as long as it works, but 1.11 seems rather old.